mu4/scores/view/scoresmodel.h

```diff
@@ -20,22 +20,46 @@
 #define MU_SCORES_SCORESMODEL_H
 
 #include <QObject>
+
 #include "modularity/ioc.h"
+#include "async/asyncable.h"
 #include "actions/iactionsdispatcher.h"
+#include "../iscoresconfiguration.h"
+#include "domain/notation/imsczmetareader.h"
+#include "ilauncher.h"
 
 namespace mu {
 namespace scores {
-class ScoresModel : public QObject
+class ScoresModel : public QObject, public async::Asyncable
```

Comment:
What about RecentScoreListModel?

Current model's name may be confusing

Reply:
let's leave it as it is, because this class is not only about the recent scores list. Here scoreы import, open and in the future there will be a list of templates.